features/zones/Index.feature

```diff
@@ -0,0 +1,54 @@
+Feature: The create Zone flow works
+  Background:
+    Given the CSS selectors
+      | Alias                               | Selector                                            |
+      | name-input                          | [data-testid="name-input"]                          |
+      | create-zone-button                  | [data-testid="create-zone-button"]                  |
+      | environment-universal-radio-button  | [data-testid="environment-universal-radio-button"]  |
+      | environment-kubernetes-radio-button | [data-testid="environment-kubernetes-radio-button"] |
+      | ingress-input-switch                | [data-testid="ingress-input-switch"]                |
+      | egress-input-switch                 | [data-testid="egress-input-switch"]                 |
+    When I visit the "/zones/create-zone" URL
+
+  Scenario: The form shows only the initial elements
+    Then the "$name-input" element exists
+    Then the "$create-zone-button" element exists
+    Then the "$create-zone-button" element is disabled
```

Comment:
Not super important but if the the "$create-zone-button" is disabled, that means it exists right? Also, is there anything that would make the `$name-input` not exist? i.e. Are we just testing Vue here?

Just as a note, when I was moving some of the old CLI tests I wanted to replicate everything that was there, even the things that when I was doing it I was thinking "this is just testing Vue".

Moving forwards I would like to try to think about only testing things we own and getting as much value as possible out of the tests by choosing what to test and what _not_ to test and getting a good balance between the two. Like these two 'exists' checks here, I'm not totally sure if there is any code of ours directly in the component that would cause these to fail? They don't seem to be using any logic at all to render/not-render them.

Anyway as I said not super important, but just wanted to communicate that here.

Reply:
> I’m removing cases of double-asserting like this.

not sure if you are gonna remove things here or not lemme know.

Reply:
I already have?

Reply:
Oh sorry was a bit difficult to tell on GH, thanks!